DigitalExcellence/dex-frontend
# Updated favicon

## Description

Update favicon to DeX logo X.

## Type of change

## Checklist


## Steps to Test or Reproduce
1. Run DeX project
2. See the new favicon

## Link to issue

Closes: #145
